CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Release 1.57.0 25-05-2021
+===============================================
+
+- update googleapis to the 05/18/2021 (#837)
+
+===============================================
```

Comment:
- Remove the `==` in line 2 and line 6.
- Capitalize the first word update for consistency.

Reply:
Done